src/main/java/com/keboola/tableexporter/Application.java

```diff
@@ -10,25 +10,27 @@
 import com.keboola.tableexporter.exception.ApplicationException;
 import com.keboola.tableexporter.exception.CsvException;
 import com.keboola.tableexporter.exception.UserException;
+import oracle.jdbc.OracleConnection;
 import org.json.JSONObject;
 
 public class Application {
 
-    private static String dbPort;
-    private static String dbHost;
-    private static String dbUser;
-    private static String dbPassword;
-    private static String dbName;
-    private static String query;
-    private static String outputFile;
-    private static String tnsnamesPath;
-    private static String tnsnamesService;
+    private String dbPort;

```

Comment:
Hodinu som hladal chybu v teste, kym som zistil, ze za to mozu tieto staticke premenne, ktore si uchovavali stav medzi testami.
Vsetci staticke veci som vyhodil.